src/Xive/Cell/FileCell.cs

```diff
@@ -68,7 +70,33 @@ private FileCell(IScalar<string> path)
         public byte[] Content()
         {
             byte[] result = new byte[0];
-            if (File.Exists(this.path.Value()))
+
+            if (this.path.Value().Equals("_guts.xml"))
+            {
+                var patch = new Directives().Add("items");
+
+
+                foreach (var file in Directory.GetFiles(
```

Comment:
Like you already mentioned: Cell is the wrong place for guts aggregation.

Reply:
Looks like I do not know how to undo things via git!